docs/E2ETESTING.md

```diff
@@ -0,0 +1,77 @@
+# End-to-End (E2E) Testing with Stark
+
+The stark starter comes with a setup to run e2e tests with protractor.
+
+The showcase app uses Travis CI and BrowserStack to automatically run e2e-tests on multiple setups (IE, Safari, Android, IOS, ...).
+
+## Protractor
+
+Protractor is an end-to-end test framework for Angular and AngularJS applications. Protractor runs tests against your application running in a real browser, interacting with it as a user would.
+
+For more information go to [https://www.protractortest.org](https://www.protractortest.org).
+
+## BrowserStack (optional)
+
+If you want to run your e2e-tests automatically on multiple setups one of the easiest solutions is BrowserStack.
+They offer a free solution for open source projects.
+
+For more information go to [https://www.browserstack.com/automate](https://www.browserstack.com/automate).
+
+## Running end-to-end tests
+
+Install all dependencies.
+
+```shell
+npm run install:all
+```
+
+Build the application
+
+```shell
+npm run build:showcase
+```
+
+Run the end-to-end tests with protractor.
+
+```shell
+npm run test:showcase:e2e
+```
+
+## Running end-to-end tests with BrowserStack
+
+### Prerequisites
+
+-   BrowserStack Account ([https://www.browserstack.com/](https://www.browserstack.com/))
+
+### Setup
+
+Find your credentials on the [BrowserStack dashboard](https://automate.browserstack.com/dashboard).
+
+Set the environmental `BROWSERSTACK_USERNAME` to the username and `BROWSERSTACK_ACCESS_KEY` to the access key.
+
+### Run the tests
+
+```shell
+npm run test:e2e:showcase:browserstack
+```
+
+_If you want to change the devices / browsers on which the tests are run you can change the `CAPABILITIES` variable in `showcase/protractor.browserstack.conf.js`. For the correct configurations see: [https://www.browserstack.com/automate/capabilities](https://www.browserstack.com/automate/capabilities)_
+
+# Adding Tests
+
+If you add tests they should be added to `src/e2e/name.e2e.ts`.
```

Comment:
"... should be added under the `src/e2e/` folder"

Reply:
I changed this to:

### Adding Tests

If you add tests they should be added as `starter/e2e/src/[testSubset]/testName.e2e-spec.ts`.
They are automatically found here by the configuration of `specs` in the `packages/stark-testing/protractor.conf.js`.

Keeping e2e tests separate like this prevents confusion when testing multiple components.